boards/riscv32/rv32m1_vega/doc/rv32m1_vega.rst

```diff
@@ -0,0 +1,782 @@
+
+===   ========   =================
+Pin   Port/bit   Function
+===   ========   =================
+2     -          SDA_GPIO0
+4     -          BRD_IO_PER
+6     -          RST_SDA
+8     -          BRD_IO_PER
+10    -          P5V_INPUT
+12    -          GND
+14    -          GND
+16    -          P5-9V VIN
+===   ========   =================
+
+**Bottom right expansion header (J4)**
+
+Note that the headers at the bottom of the board have odd-numbered
+pins on the top, unlike the headers at the top of the board.
+
+Odd/top pins:
+
+===   ========   ========================================
+Pin   Port/bit   Function
+===   ========   ========================================
+1     -          TAMPER2
+3     -          TAMPER1/RTC_CLKOUT
+5     -          TAMPER0/RTC_WAKEUP_b
+7     E/2        ADC0_SE19
+9     E/5        LPCMP1_IN2/LPCMP1_OUT
+11    -          DAC0_OUT/ADC0_SE16/LPCMP0_IN3/LPCMP1_IN3
+===   ========   ========================================
+
+Even/bottom pins:
+
+===   ========   ===========================================
+Pin   Port/bit   Function
+===   ========   ===========================================
+2     C/11       ADC0_SE6
+4     C/12       ADC0_SE7
+6     B/9        ADC0_SE3
+8     E/4        ADC0_SE21
+10    E/10       ADC0_SE19 (and E/10, I2C_SDA via 0 Ohm DNP)
+12    E/11       ADC0_SE20 (and E/11, I2C_SCL via 0 Ohm DNP)
+===   ========   ===========================================
+
+Additional Pins
+~~~~~~~~~~~~~~~
+
+For an up-to-date description of additional pins (such as buttons,
+LEDs, etc.) supported by Zephyr, see the board DTS files in the Zephyr
+source code, i.e.
+:file:`boards/riscv32/rv32m1_vega/rv32m1_vega_ri5cy.dts` for RI5CY.
+
+See the schematic in the documentation available from the `OpenISA
+GitHub releases`_ page for additional details.
+
+System Clocks
+=============
+
+The RI5CY core is configured to use the slow internal reference clock
+(SIRC) as the clock source for an LPTMR peripheral to manage the
+system timer, and the fast internal reference clock (FIRC) to generate
+a 48MHz core clock.
+
+Serial Port
+===========
+
+The USB connector at the top left of the board (near the RESET button)
+is connected to an OpenSDA chip which provides a serial USB
+device. This is connected to the LPUART0 peripheral which the RI5CY
+core uses by default for console and logging.
+
+.. warning::
+
+   The OpenSDA chip cannot be used to flash or debug the RISC-V cores.
+
+   See the next section for flash and debug instructions for the
+   RISC-V cores using an external JTAG dongle.
+
+Programming and Debugging
+*************************
+
+.. _rv32m1-programming-hw:
+
+.. important::
+
+   To use this board, you will need:
+
+   - a `SEGGER J-Link`_ debug probe to debug the RISC-V cores
+   - a J-Link `9-Pin Cortex-M Adapter`_ board and ribbon cable
+   - the SEGGER `J-Link Software and Documentation Pack`_ software
+     installed
+
+   A JTAG dongle is not included with the board itself.
```

Comment:
Is this really the only way? Anyone who wants to use this board needs to spend hundreds of dollars on a JTAG debugger?

Reply:
> The [open-isa website](http://open-isa.org/get-started/) implies that everything you need is included in the box. Is there no other way to use the board besides buying a specialised JTAG debugger?

There is no other way.